js/twister_newmsgs.js

```diff
@@ -101,7 +101,13 @@ function requestMentionsCount() {
         $.MAL.soundNotifyMentions();
 
         if ($.Options.getShowDesktopNotifMentionsOpt() === 'enable') {
-            $.MAL.showDesktopNotif(false, polyglot.t('You got')+' '+polyglot.t('new_mentions', _newMentions)+'.', false,'twister_notification_new_mentions', $.Options.getShowDesktopNotifMentionsTimerOpt(), openMentionsModal, false)
+            $.MAL.showDesktopNotif(false, polyglot.t('You got')+' '+polyglot.t('new_mentions', _newMentions)+'.', false,'twister_notification_new_mentions', $.Options.getShowDesktopNotifMentionsTimerOpt(), function() {
+                if (window.location.pathname === '/home.html' || window.location.pathname === '/following.html' ) {
```

Comment:
There must be a better way to tell which current page is instead of relying on absolute filename. This will also probably break support for subdirectory hosting that was recently added I think.